tests-web/www/js/layers.js

```diff
@@ -66,6 +66,25 @@ export const makeLanePolygonLayer = (text) => {
 
   return new L.geoJSON(JSON.parse(text), {
     style: function (feature) {
+      if (feature.properties.type == "Footway") {
```

Comment:
The styling is not great. Any ideas for distinguishing the two?

Reply:
Maybe it's time to start copying design from the [strassenraumkarte-neukoelln](https://github.com/SupaplexOSM/strassenraumkarte-neukoelln) :P

Reply:
I already lifted the dashed outline idea and path color from there ;)